adr/recipes/skipped_tests.py

```diff
@@ -8,19 +8,19 @@
 """
 from __future__ import absolute_import, print_function
 
+from ..context import override
 from ..query import run_query
 
+# RUN_CONTEXTS = {'limit': {'default': 25,
+#                           'help': "Maximum number of users in result"},
+#                 'sort_key': {'default': 1,
+#                              'help': "Key to sort on (int, 0-based index)",
+#                              }
+#                 }
```

Comment:
Remove this commented out block.

Reply:
I removed in the second commit yesterday, but thank anyway, I often forgot to delete commented code :)